musicdiff/m21utils.py

```diff
@@ -332,7 +340,10 @@ def get_tuplets_type(note_list):
             for i, note_tuple in enumerate(tuplets_list):
                 if len(note_tuple) > ii:
                     if note_tuple[ii] == "start":
-                        assert start_index is None
+                        # Some medieval music has weirdly nested triplets that
```

Comment:
So what is happening in this case? Are we losing information about the tuplet that starts first? This may still be ok, just to make things work, but maybe we want to print a warning, in case this is not the wanting behavior

Reply:
We're not losing any information, and if the other score has a 'start' here, it will show up as a difference.  musicdiff is not trying to be a "check for well-formedness" tool, just a diff tool.

Reply:
Right!